src/test/ui/traits/vtable/vtable-diamond.stderr

```diff
@@ -0,0 +1,56 @@
+error: Vtable Entries: [
+    MetadataDropInPlace,
+    MetadataSize,
+    MetadataAlign,
+    Method(
+        DefId(0:4 ~ vtable_diamond[4564]::A::foo_a),
+        [
+            S,
+        ],
+    ),
```

Comment:
I wonder why `Method` isn't using `Instance`, but I'm mostly saying this for the printing quality.

Reply:
Done.